week1/index.js

```diff
@@ -1,25 +1,42 @@
-// To run this file: node index.js
-// Then in your browser: http://localhost:8080
-var http = require('http');
+const hostname = '127.0.0.1';
+const port = 3000;
+var counter = 10;
 
-var port = 8080;
+const server = http.createServer((request, response) => {
+    response.statusCode = 200;
+    response.setHeader('Content-Type', 'text/html');
 
-var server = http.createServer();
+    console.log('The new received http request is: ', request.url)
 
-// Start the HTTP server, start listening for requests
-server.listen(port, function(error) {
-  if (error) {
-    console.log(error);
-  } else {
-    console.log('api listening on port', port);
-  }
+    switch (request.url) {
+        case '/state':
+        case '/':
+            console.log('Original counter value:', counter, '\n')
```

Comment:
Try something:
1. Go to `/state`
2. Go to `/add`
3. Go back to `/state` (as a new request, not by hitting the back button

What gets logged to the console? Is that correct/true?